Gemfile.lock

```diff
@@ -20,6 +20,7 @@ GEM
     parallel (1.20.1)
     parser (3.0.1.1)
       ast (~> 2.4.1)
+    prettier_print (1.2.1)
```

Comment:
curious about why this upgrade to prettier_print when 1.2.0 is required by syntax_tree...

Reply:
Ah so this was automatically set when I added in the `syntax_tree` gem and ran `bundle install`. Since `syntax_tree` for version `6.1.1` has `>= 1.2.0` set as a dependency, I'm assuming that `1.2.1` is the highest version that meets that requirement.